Source/Core/Common/HttpRequest.cpp

```diff
@@ -33,7 +33,8 @@ class HttpRequest::Impl final
   void FollowRedirects(long max);
   s32 GetLastResponseCode();
   Response Fetch(const std::string& url, Method method, const Headers& headers, const u8* payload,
-                 size_t size, AllowedReturnCodes codes = AllowedReturnCodes::Ok_Only);
+                 size_t size, AllowedReturnCodes codes = AllowedReturnCodes::Ok_Only,
+                 const std::vector<Multiform>& multiform = {});
```

Comment:
Can this be a `std::span`? Same for the rest of the functions you pass `multiform` through.